src/command_modules/azure-cli-search/azure/cli/command_modules/search/_params.py

```diff
@@ -0,0 +1,18 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+# pylint: disable=line-too-long
+from azure.cli.core.commands.parameters import name_type
+
+
+def load_arguments(self, _):
+    with self.argument_context('search services') as c:
```

Comment:
services should be singular. `az search service create`.

Reply:
fixed